### PR TITLE
Add docfile record only if the file exists - V32

### PR DESCRIPTION
**Pull request by mrodozov**
please test

### CODE REVIEW

File: cmsBuild [hunk 1]
```diff
@@ -3614,18 +3614,20 @@ def buildSpecs(topLevelPackages):
     if scheduler.brokenJobs:
         sys.exit(1)
 
-def getPackageDependencies(pkg):
+def getPackageDependencies(pkg, opts):
```
Comment (smuzaffar) [inline cmsBuild]: @mrodozov , you do not need to pass `opts` here. pkg object should have options

**Comment by smuzaffar** [comment]
can you please update V00-33 branch too, I have merged that change already :-(

**Comment by mrodozov** [comment]
yes, there I just updated the branch and you merged it in the same instant 😅

**Comment by mrodozov** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

**Pull request merged**